File: TextosJapones/src/main/java/org/jisho/textosJapones/components/listener/VinculoServiceListener.java
package org.jisho.textosJapones.components.listener;

import javafx.collections.ObservableList;
import org.jisho.textosJapones.model.entities.VinculoPagina;

public interface VinculoServiceListener {

	ObservableList<VinculoPagina> getVinculados();

	ObservableList<VinculoPagina> getNaoVinculados();
	
}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/components/listener/VinculoServiceListener.kt
package br.com.fenix.processatexto.components.listener

import br.com.fenix.processatexto.model.entities.processatexto.VinculoPagina
import javafx.collections.ObservableList

interface VinculoServiceListener {
    val vinculados: ObservableList<VinculoPagina>
    val naoVinculados: ObservableList<VinculoPagina>
}